src/docs/releases/2.1.0.md

```diff
@@ -65,14 +65,16 @@ In the Roles feature, there were previously `AssignRoles` and `AssignRole_{RoleN
 
 #### Roles Now Include Types
 
-A new `Type` property has been added to the `IRole` interface, which allows defining a role's behavior:
+A new `Type` property has been added to the `IRole` interface, and partially surfaced on the role editor UI, which allows defining a role's behavior:
 
 - **`Standard`**: A typical role that allows granular permission assignment.
 - **`System`**: Non-removable system roles such as 'Anonymous' and 'Authenticated.'
 - **`Owner`**: A special role type that automatically grants all permissions without needing to specify them. This type is designed for roles like `Administrator` or `SiteOwner`.
 
-!!! note
-    When a user with a role marked as `Owner` logs in, role-specific claims are excluded from the claims principal, as full access is already granted.
+Furthermore, the `SiteOwner` permission has been deprecated and will be removed in future releases. To prevent any breaking changes, we've introduced a fast-track migration that automatically assigns the `Owner` role type to any role with the `SiteOwner` permission.
+
```

Comment:
```suggestion

Instead of authorizing against the `SiteOwner` permission, the Recipes feature now uses its own  Manage Recipes permission (which is automatically available for the Administrator role).

```